app/objects/c_source.py

```diff
@@ -15,6 +15,7 @@ def __init__(self, name, facts):
         super().__init__()
         self.name = name
```

Comment:
add rules to init

Reply:
def __init__(self, name, facts):

to

def __init__(self, name, facts, rules):